src/Rector/Set/CakePHPSetList.php

```diff
@@ -62,6 +62,16 @@ final class CakePHPSetList implements SetListInterface
      */
     public const CAKEPHP_44 = __DIR__ . '/../../../config/rector/sets/cakephp44.php';
 
+    /**
+     * @var string
+     */
+    public const CHRONOS_3_DATE = __DIR__ . '/../../../config/rector/sets/chronos3-date.php';
+
+    /**
+     * @var string
+     */
+    public const CHRONOS_3_DATETIME = __DIR__ . '/../../../config/rector/sets/chronos3-datetime.php';
```

Comment:
I think we'd want a combined chronos 3 set to upgrade from 2.x to 3.x.

Reply:
We can add that after this. This is already a huge improvement.